app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+    :recoverable, :rememberable, :validatable
```

Comment:
I may be splitting hairs a bit here seeing as I know this will be the next ticket, but I wouldn't normally want to add the recoverable module in this commit/PR, I'd want it in the chunk of work we're doing that section of work

Reply:
I don't think this is splitting hairs – it's entirely reasonable, as we want the meaning in the next piece of work's commits "we had to add the `:recoverable` module to achieve this" to be present there, not here.

Reply:
Removed